mandatory/3-playing-computer.js

```diff
@@ -7,12 +7,18 @@
   Answer the following questions:
 
   1. This program throws an error. Why? (If you can't find it, try executing it).
+  b is not defined
   2. Remove the line that throws the error.
   3. What is printed to the console?
+  2 6 4 9 6 13 8 
   4. How many times is "f1" called?
+  2 times
   5. How many times is "f2" called?
+   3 times
   6. What value does the "a" parameter take in the first "f1" call?
+  7
```

Comment:
The correct answer here is `1` as explained at the bottom of this document https://github.com/CodeYourFuture/JavaScript-Core-1-Coursework-Week2-Solution/blob/main/mandatory/solutions.md